packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -72,4 +76,84 @@ MeetingsUtil.checkForCorrelationId = (deviceUrl, locus) => {
   return false;
 };
 
+
+/**
+ * Will check to see if specified media codec is supported.
+ * @param {String} codec Media codec.
+ * @returns {Boolean}
+ */
+MeetingsUtil.hasMediaCodec = (codec) => {
+  let hasCodec = false;
+  const video = window.document.createElement('video');
+  const PROBABLY = 'probably';
+
+  if (window.MediaSource) {
+    hasCodec = window.MediaSource.isTypeSupported(codec);
+  }
+  else {
+    hasCodec = video.canPlayType(codec) === PROBABLY;
```

Comment:
Modenizer does this exact check to determine media capabilities of a browser. `canPlayType` will return only three values 'probably', 'maybe', or ''. If the return is not 'probably' then it doesn't support the codec. We use this as a fallback, preferring `isTypeSupported` first.

Reply:
I have a Firefox browser where it fails connecting to call due to H264 issue.  I see same values on browser which supports H264 and on browser which doesn't support H26 too.

window.MediaSource.isTypeSupported('video/mp4; codecs="avc1.42E01E"') - returns true 
video.canPlayType('video/mp4; codecs="avc1.42E01E"') - returns "probably".

Reply:
That means the browser you think doesn't support H264 actually does, it supports H.264 Baseline. There's Baseline, Main and High. Let's see which one it doesn't support.

Run this for me, this is for H.264 High:
`window.MediaSource.isTypeSupported('video/mp4; codecs="avc1.64001E"');`

then run this:
`new RTCPeerConnection().createOffer({offerToReceiveVideo: true}).then( offer => console.log(offer.sdp) )`